lib/backend/helpers.go

```diff
@@ -61,3 +61,38 @@ func ReleaseLock(ctx context.Context, backend Backend, lockName string) error {
 	}
 	return nil
 }
+
+// ResetLockTTL resets the TTL on a given lock.
+func ResetLockTTL(ctx context.Context, backend Backend, lockName string) error {
+	if lockName == "" {
+		return trace.BadParameter("missing parameter lock name")
+	}
+
+	key := []byte(filepath.Join(locksPrefix, lockName))
+
+	item, err := backend.Get(ctx, key)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if _, err := backend.Put(ctx, *item); err != nil {
+		return trace.Wrap(err)
+	}
+
+	return nil
+}
+
+// RunWhileLocked allows you to run a function while a lock is held.
+func RunWhileLocked(ctx context.Context, backend Backend, lockName string, ttl time.Duration, fn func() error) error {
```

Comment:
If we're going to introduce the concept of lock refreshes, this function should handle them internally.

Reply:
Handles them internally now